django/db/models/expressions.py

```diff
@@ -1501,11 +1503,19 @@ class OrderBy(Expression):
     template = "%(expression)s %(ordering)s"
     conditional = False
 
-    def __init__(
-        self, expression, descending=False, nulls_first=False, nulls_last=False
-    ):
+    def __init__(self, expression, descending=False, nulls_first=None, nulls_last=None):
         if nulls_first and nulls_last:
             raise ValueError("nulls_first and nulls_last are mutually exclusive")
+        if nulls_first is False or nulls_last is False:
+            # When the deprecation ends, replace with:
+            # raise ValueError(
+            #     "nulls_first and nulls_last values must be True or None."
+            # )
+            warnings.warn(
+                "Passing nulls_first=False or nulls_last=False is deprecated, use None "
+                "instead.",
+                RemovedInDjango50Warning,
+            )
```

Comment:
Should've asked this on the first pass, sorry...
I can never remember how stack level plays out. Does this currently highlight _this_ frame, or the one where _the user_ passed through `nulls_first/last=False`? I'd _presume_ the latter is the ideal?

Reply:
Good point, I added `stacklevel`.